### PR TITLE
Change link on map inside iframe

### PR DESCRIPTION
**Pull request by chizhovm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "watch-test": "karma start --auto-watch --no-single-run",
     "pack": "cross-env NODE_ENV=production webpack -p --progress",
     "pack-nightly": "NODE_ENV=nightly webpack --progress",
-    "start": "cross-env NODE_ENV=development webpack-dev-server --progress --host 0.0.0.0 --hot --https"
+    "start": "webpack-dev-server --mode development --watch"
```
Comment (rizomaa) [inline package.json]: @chizhovm Please comment how the changes relate to this task ?

File: src/constants/URLs.js [hunk 2]
```diff
@@ -28,7 +28,8 @@ const OFDB_API_LINK = (() => {
 
     default:
       // production  // ?? api.ofdb.io ??
-      return "https://kartevonmorgen.org/api/";
+      // return "https://kartevonmorgen.org/api/";
+      return 'https://cors-anywhere.herokuapp.com/' + "https://kartevonmorgen.org/api/";
```
Comment (rizomaa) [inline src/constants/URLs.js]: and this?

File: package.json (hunk 1 above)
Comment (chizhovm) [reply package.json]: This change relate https://github.com/diglabby/mapa/issues/340. I think it's my mistake to introduce it into this PR. Without this change I couldn't start the project.

Comment (rizomaa) [reply package.json]: @chizhovm please move the changes out of the commit. 
They are very important but they relates to another issue. 

So please think we push into Pull Request the only code we need to solve the issue. For all other code we can open a new issue and do the same. Such a division helps to monitor code changes perfectly.

File: src/constants/URLs.js (hunk 2 above)
Comment (rizomaa) [reply src/constants/URLs.js]: @chizhovm what about the code?

**Comment by chizhovm** [comment]
I deleted these changes and made new commit.

File: src/components/MapOptions/IframeForm/IframeForm.jsx [hunk 3]
```diff
@@ -28,9 +28,9 @@ const IframeForm = ({ data, close }) => {
   const txtWidth = 'width="' + width
   const txtHeight = '" height="' + height
   const endIframe = '" ></iframe>'
-  const href = '<a href="https://mapa.falanster.by/" target="_blank">The map</a>'
+  // const href = '<a href="https://mapa.falanster.by/">The map</a>'
```
Comment (rizomaa) [inline src/components/MapOptions/IframeForm/IframeForm.jsx:31]: @chizhovm чаму тут спратрэбілася закаментаваць?

**Comment by chizhovm** [comment]
issue_256
This link is outside the widget.
I commented it out, but it's better to remove it

**Comment by chizhovm** [comment]
I commented it out, but it's better to remove it

**Comment by rizomaa** [comment]
@chizhovm дзякуй за працу. прымаю ПР

**Pull request merged**